cpp/src/stream_compaction/drop_duplicates.cu

```diff
@@ -226,8 +226,89 @@ std::unique_ptr<experimental::table>
   return detail::gather(input, unique_indices_view, false, false, false, mr, stream);
 }
 
+/**---------------------------------------------------------------------------*
+ * @brief A structure to be used for checking `NAN` at an index in a 
+ * `column_device_view`
+ *
+ * @tparam T The type of `column_device_view`
+ *---------------------------------------------------------------------------**/
+template <typename T>
+struct check_for_nan
+{
+  /**---------------------------------------------------------------------------*
+   * @brief Construct a strcuture
```

Comment:
You copied a typo, I think it should be
```suggestion
   * @brief Construct a structure
```
but it's not very useful comment either way :)

Reply:
This code is moved from `.hpp` file. Since it had `__device__` i moved to `.cu` file. I will review the doc more.